velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -140,14 +140,42 @@ void toDuckDbFilter(
       }
       break;
     }
+    case common::FilterKind::kBigintValuesUsingHashTable: {
+      auto valuesFilter =
+          static_cast<common::BigintValuesUsingHashTable*>(filter);
+      const auto& values = valuesFilter->values();
+      if (values.size() == 1) {
+        filters.PushFilter(colIdx, constantEqualFilter(*values.begin()));
+      } else {
+        auto duckFilter = std::make_unique<::duckdb::ConjunctionOrFilter>();
```

Comment:
I wonder if it would be more efficient to use COMPARE_IN filter. CC: @Mytherin

Reply:
Yeah, I'm revising to use COMPARE_IN filter. (Didn't find how to construct it with the old version. Now I'm able to)